server/protocols/jmap-draft/src/test/java/org/apache/james/jmap/draft/methods/SetMessagesCreationProcessorTest.java

```diff
@@ -174,13 +173,13 @@ public void setUp() throws MailboxException, DomainListException, UnknownHostExc
         
         fakeSystemMailboxesProvider = new TestSystemMailboxesProvider(() -> optionalOutbox, () -> optionalDrafts);
         session = MailboxSessionUtil.create(USER);
-        MIMEMessageConverter mimeMessageConverter = new MIMEMessageConverter(mock(AttachmentContentLoader.class));
-        messageAppender = new MessageAppender(mockedMailboxManager, mockMessageIdManager, mockedAttachmentManager, mimeMessageConverter, JMAPConfiguration.builder().enable().build());
+        MIMEMessageConverter mimeMessageConverter = new MIMEMessageConverter(mock(BlobManager.class));
```

Comment:
The mock for `BlobManager` is exists, we can reuse it.
Maybe one instance is better.

Reply:
I succeeded to drop the mock in https://github.com/apache/james-project/pull/584